cfgov/unprocessed/js/modules/transition/AlphaTransition.js

```diff
@@ -43,28 +45,37 @@ function AlphaTransition( element ) {
   }
 
   /**
-   * Remove all transition classes.
+   * Remove all transition classes, if transition is initialized.
+   * @returns {boolean}
+   *   True, if the element's CSS classes were touched, false otherwise.
    */
   function remove() {
-    _baseTransition.remove();
+    return _baseTransition.remove();
   }
 
   /**
    * Add a "transition-duration: 0" utility CSS class.
+   * @returns {boolean}
+   *   Whether the element's CSS classes where touched or not.
+   *   False means the transition has not been initialized.
    */
   function animateOn() {
-    _baseTransition.animateOn();
+    return _baseTransition.animateOn();
   }
 
   /**
    * Remove a "transition-duration: 0" utility CSS class.
+   * @returns {boolean}
+   *   Whether the element's CSS classes where touched or not.
+   *   False means the transition has not been initialized.
    */
   function animateOff() {
-    _baseTransition.animateOff();
+    return _baseTransition.animateOff();
```

Comment:
Hmm, that's a good question. I don't think so in this case. What this method is doing is effectively turning the transition animation off by adding a class to the dom that sets the duration to 0, so `animateOff` describes the action that happens, which is different from something that checks the state, like `isExpanded`.

Calling this method when the associated dom element was not initialized would look the same as when it was initialized (returning undefined), so as a convenience to testing, I updated it to return true when the dom is touched, or false if it's not. However, that's not the purpose of the method (like `isDomTouched` or something), so it could return something else, like `undefined` in all cases (like classList.remove(…) does), or return the instance so the methods can be chained. But a boolean seemed to make the most sense—implying animate off either happened or didn't.